bin/check-asg-instances-created.rb

```diff
@@ -78,9 +78,9 @@ def run
     time_utc_offset = time_now - time_now.utc_offset
 
     if !config[:warning_limit].nil?
-      warning = config[:warning_limit]
+      warning = config[:warning_limit].to_i
     elsif !config[:critical_limit].nil?
-      critical = config[:critical_limit]
+      critical = config[:critical_limit].to_i
```

Comment:
I think what we really want here is this:
```ruby
option :warning_limit,
       proc: proc(&:to_i),
```

and then later we can if we wish to we can check if its zero:
```ruby
unknown "you passed critical_limit: #{config[:critical_limit]} which evaluated as 0" if config[:critical_limit].zero?
```